tests/end-to-end/backup/pkg/tests/servicecatalog/app_broker.go

```diff
@@ -211,6 +230,58 @@ func (f *appBrokerFlow) createApplication() error {
 	return err
 }
 
+// createChannel creates a Knative channel that would normally be installed by the application controller upon helm-chart installation. As we skip the installation part,
+// we need to ensure that the channel exists as it is required by the knative broker.
+func (f *appBrokerFlow) createChannel() error {
```

Comment:
for an app broker does it makes sense to create channel? shouldn't this be done by the httpsource, right?